instance/tests/api/test_instance.py

```diff
@@ -156,3 +163,19 @@ def test_get_log_entries(self):
         for expected_entry, log_entry in zip(expected_list, log_entries):
             self.assertEqual(expected_entry['level'], log_entry['level'])
             self.assertEqual(expected_entry['text'].format(inst_id=instance.ref.pk), log_entry['text'])
+            self.assertEqual(expected_entry['text'].format(inst_id=instance.ref.pk), log_entry['text'])
+
+    def test_get_app_servers_list(self):
+        """
+        GET - App Servers
+        """
+        self.api_client.login(username='user3', password='pass')
+        instance = OpenEdXInstanceFactory(name="Test!")
+        instance.logger.info("info")
+        instance.logger.error("error")
+
+        response = self.api_client.get('/api/v1/instance/{pk}/app_servers/'.format(pk=instance.ref.pk))
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        self.assertTrue('app_servers' in response.data)
```

Comment:
Can we add a test or assertion that actually checks the format of the data returned by this API?

Reply:
Done. I also improved other tests, one of them was wrong because it didn't create appservers in the instance. Now I create e.g. 7 appservers and check that the API returns less (e.g. 5).